packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -3579,5 +3546,5 @@ typedef DiagnosticsNodesFilter = List<DiagnosticsNode> Function(List<Diagnostics
 /// A callback that returns a [DiagnosticsNodeDelegateGetter] for adding `node`
```

Comment:
This callback returns a DiagnosticsNodeDelegate not a DiagnosticsNodeDelegateGetter